_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -0,0 +1,162 @@
+
+- **Tooling**: There is never one way to do something, so try something else. Switch text editor for a month, or explore alternative methods of doing the same thing.
+
+You can use this knowledge to start offering useful advice that can start to influence your team, for example could be in the form of trying few new idea and offering feedback on their tradeoffs.
+
+## Helping others
+
+You can help your team out by using some of the skills from above.  You can then start thinking of expanding your influence within the company. By being a programmer you already have skills that a lot of people would like to learn or have a better working knowledge of it.
+
+If you’ve got this far in the article, it’s very likely you’ve got enough skills to lead workshops internally, and to write blog posts about your experiences to help other developers at your level. For example, this month I’ve ran a workshop on keyboard shortcuts and on learning to program in Swift. Both of them required maybe 2 hours of preparation, and an email or two announcing that it’s happening.
+
+Other options are to give technical talks within your company on specific topics, for example [Licensing for OSS code][11] is a talk I gave internally to Artsy. We now have a weekly team “Lunch & Learn” where we give anyone the chance to talk, or request a talk on a topic. 
+
+This gives a lot of space for personal growth too, as these are really easy to transition into blog posts and meetup talks.
+
+Finally, try to pair with programmers outside of your direct team, it could be on whatever they’re working on - you’ll learn a bit more about other systems and they’ll get to [rubber duck][12] their problems.
+
+## Side Projects
+
+I’ve tried to focus this post specifically on things you can do on work time. Not everyone has the ability to go home and spend a few hours on this and that. However, programming at the early stage can generally be a “you get out what you put in” kind of deal with time.  
+
+Side projects give you the chance to test out new ideas in an isolated environment that is totally under your control. They are great places for exploring what makes a system tick. Some side-projects are built to be [thrown away][13] others can [to last][14] [for years][15]. Both are valuable spaces for experimentation.
+
+## Contributing to Open Source
+
+A lot of people’s work relies on Open Source code, as you are starting to branch out into having a larger impact - perhaps making smaller improvements to the projects you rely on every day could help. This is [how I got started][16] with larger projects in the Open Source world.
+
+Working in these projects exposes you to whole new teams of people with, hopefully, very different perspectives. It will change you, you will change them.
+
+## Wrap-up
+
+There is no simple “one-step to increasing your impact” article. Only a collection of ideas that you can re-apply in your own context. 
```

Comment:
So this is interesting. Does this mean that furthering in your career and developing yourself as an engineer is purely on the impact scale / ladder? 

Yes, the more knowledge you consume and the more things you learn, the more great things you can do on your project and with your team. One could call this impact, but to me, impact goes much much further.

Essentially, I'm worried that using the word 'impact' a lot in this paragraph, without occasionally swapping it for 'self-development' or something, makes it sound like you need to go out and blog / speak / OSS / do things beyond your company in order to "level up" as it were. Whilst these obviously help (as the article rightly points out!), I think indeed some of your other approaches (research, personal side projects, etc) are also great and may be lost under the blanket term 'impact'. 

Does this make sense?